packages/ActionMenu/src/action-menu.scss

```diff
@@ -8,15 +8,25 @@
   /* Default */
   --ids-action-menu-font-family: #{tokens.$primary-font-family};
   --ids-action-menu-font-size: #{tokens.$font-size-4};
+
+  --ids-action-menu-helper-text-size: #{tokens.$font-size-2};
+  --ids-action-menu-helper-text-color: #{tokens.$grey-600};
 }
 
 [data-brand="workleap"] {
-  --ids-action-menu-font-family:  var(--hop-body-sm-font-family);
+  --ids-action-menu-font-family: var(--hop-body-sm-font-family);
   --ids-action-menu-font-size: var(--hop-body-sm-font-size);
+
+  --ids-action-menu-helper-text-size: var(--hop-body-xs-font-size);
+  --ids-action-menu-helper-text-color: var(--hop-neutral-text-weak);
```

Comment:
We should be using Igloo's HelperText component. These properties are already set in there.